### PR TITLE
cling example was wrong

### PR DESCRIPTION
**Pull request by couet**

### CODE REVIEW

File: manual/cling/index.md [hunk 1]
```diff
@@ -127,18 +127,16 @@ filename     line:size busy function type and name
...
 root [] .>
-root [] ?
 {% endhighlight %}
 
 Here we see:
 
 -   Use `.class` as quick help and reference
--   Unix like I/O redirection using `.> out.txt` and unredirection with `.>`
--   Use `?` to get help on all ‘‘raw'' interpreter commands
--   Use @ to abort a multi-line command
```
Comment (ferdymercury) [inline manual/cling/index.md:141]: This one is still valid, by changing to `.@`

Comment (couet) [reply manual/cling/index.md:141]: This PR is correct now. Do you agree ? I checked it . the TLine `l`was not not created either....
Do you agree I merge it ?

Comment (ferdymercury) [reply manual/cling/index.md:141]: I would restore line 141, but changing @ with .@
.@ works well for interrupting a multi-line command.

Comment (couet) [reply manual/cling/index.md:141]: I remove it because it is better explained in the next example

Comment (ferdymercury) [reply manual/cling/index.md:141]: oh I see.

And what about removing the number within the root[ ] brackets?

File: manual/cling/index.md [hunk 2]
```diff
@@ -127,18 +127,16 @@ filename     line:size busy function type and name
 (compiled)     (NA):(NA) 0 public: static int ImplFileLine();
 (compiled)     (NA):(NA) 0 public: static const char *ImplFileName();
 (compiled)     (NA):(NA) 0 public: static int DeclFileLine();
+root [] TLine l;
 root [] .> test.log
-root [] l.Dump();
+ l.Dump();
```
Comment (ferdymercury) [inline manual/cling/index.md:132]:
```suggestion
root [] l.Dump();
```

**Pull request merged**